torch/autograd/profiler.py

```diff
@@ -1519,18 +1519,28 @@ def add_column(padding, text_dir='>'):
 
     def auto_scale_flops(flops):
         flop_headers = [
-            'FLOPS',
-            'KFLOPS',
-            'MFLOPS',
-            'GFLOPS',
-            'TFLOPS',
-            'PFLOPS',
+            'FLOPS/s',
```

Comment:
According to [wiki](https://en.wikipedia.org/wiki/FLOPS), FLOPS is already a measure of floating point operations per second, maybe changing it to "flop/s" is clearer?